test/helpers/rand/name.go

```diff
@@ -0,0 +1,55 @@
+package rand
```

Comment:
don't we want to have `//go:build e2e` here aswell, like in all e2e testing packages 🤔

Reply:
true, fixed it